blog/2022-07-06-whats-new-july-2022/index.mdx

```diff
@@ -28,7 +28,7 @@ Trackpad fans rejoice; you can now pinch to zoom in and out to navigate the edit
 
 ## Nesting typegen files
 
-You’re missing out if you’re not using [our XState VS Code extension](https://marketplace.visualstudio.com/items?itemName=statelyai.stately-vscode) yet! After a top tip from our friend Erik Rasmussen, Anders added the feature to our VS Code extension. Read [Anders’ blog post on nesting XState typegen files](/blog/2022-06-13-nesting-typegen-files/index.mdx) for more details.
```

Comment:
Updated the URL here to point to the new slug, but it's hard to spot in all the formatting noise